DirectXTK_Desktop_2017.vcxproj

```diff
@@ -493,31 +493,31 @@
     <ProjectGuid>{E0B52AE7-E160-4D32-BF3F-910B785E5A8E}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>DirectXTK</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
```

Comment:
You should use ``DirectXTK_Desktop_2019.vcxproj`` instead which already has these changes.